internal/commands/config_test.go

```diff
@@ -57,39 +62,11 @@ func testConfigCommand(t *testing.T, when spec.G, it spec.S) {
 			output := outBuf.String()
 			h.AssertContains(t, output, "Interact with Pack's configuration")
 			h.AssertContains(t, output, "Usage:")
-			for _, command := range []string{"trusted-builders", "run-image-mirrors"} {
+			for _, command := range []string{"trusted-builders", "run-image-mirrors", "default-builder"} {
 				h.AssertContains(t, output, command)
 			}
 		})
 	})
-
-	when("trusted-builders", func() {
```

Comment:
I moved these tests to `config_trusted_builder_test.go`, which made more semantic sense than testing them here, where we only need to know that the appropriate commands show up.